tests/installation/setup_online_repos.pm

```diff
@@ -20,6 +20,16 @@ sub run() {
     # ordered according to real repos lists
     my @default_repos = qw(update-non-oss update-oss main-non-oss main-oss debug-main untested-update debug-update source);
 
+    if (get_var('VERSION', '') !~ /42\.[1-2]/) {
```

Comment:
Can't we assert_screen [qw(setup_online_repos-configure setup_online_repos-configure-text online-repos)], 200 and then work on last_match_has_tag instead of this VERSION hack?

I would find it clearer